java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -595,6 +597,30 @@ public CompletableFuture<Long> zcard(@NonNull String key) {
         return commandManager.submitNewCommand(Zcard, new String[] {key}, this::handleLongResponse);
     }
 
+    @Override
+    public CompletableFuture<String> zrandmember(@NonNull String key) {
```

Comment:
we could potentially overload all of these with the same name (`zrandmember`). If we did this `zrandmemberWithCountWithScores` could become `zrandmember(String key, long count, bool withScores)`

Reply:
Return values are different for each. depending on the boolean withScores you revieve a different return type.

Reply:
Ah right